deskpro/deskpro-admin-frontend
# Feature/9139 problem incidents validation

Test steps:
- Go to http://localhost:3000/#/tickets/problems
- Remove all permissions in **‘View’ Permissions** page section.
- And then move focus to another field

Expected behavior:
- An error message `Permissions field must have at least 1 items.`
